CONTRIBUTING.md

```diff
@@ -1,8 +1,75 @@
+etiquette employed by the core gno team. It lays out coding styles, simple how-to guides and tools to get you up and
+running and pushing code.
+
+If you are unsure about something, please don’t hesitate to reach out for help by opening an issue or discussion.
+Likewise, if you have an idea on how to improve this guide, go for it as well.
+
+## Table of Contents
+
+- [Important Resources](#important-resources)
+- [Getting Started](#getting-started)
+    - [Environment](#environment)
+    - [Local Setup](#local-setup)
+    - [Testing](#testing)
+    - [Repository Structure](#repository-structure)
+- [How do I?](#how-do-i)
+    - [How do I submit changes?](#how-do-i-submit-changes)
+    - [How do I report a bug?](#how-do-i-report-a-bug)
+    - [How do I request a feature?](#how-do-i-request-a-feature)
+- [Style Guides](#style-guides)
+    - [Git Commit Messages](#git-commit-messages)
+    - [Go Style Guide](#go-style-guide)
+    - [Documentation Style Guide](#documentation-style-guide)
+- [Additional Notes](#additional-notes)
+    - [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+    - [Notable Contributions](#notable-contributions)
+
+## Important Resources
+
+- **[Discord](https://discord.gg/YFtMjWwUN7)** - we are very active on Discord. Join today and start discussing all
+  things gno with fellow engineers and enthusiasts.
+- **[Active Staging](https://gno.land/)** - use the currently available staging environment to play around with a
+  production network. If you want to interact with a local instance, refer to the [Local Setup](#local-setup) guide.
+- **[Twitter](https://twitter.com/_gnoland)** - follow us on Twitter to get the latest scoop
+- **[Telegram](https://t.me/gnoland)** - join our official Telegram group to start a conversation about gno
+- **[Awesome GNO](https://github.com/gnolang/awesome-gno)** - check out the list of compiled resources for helping you
+  understand the gno ecosystem
+
+## Getting Started
+
+### Environment
+
+The gno repository is primarily based on Golang (Go), and Gnolang (Gno).
+
+The primary tech stack for working on the repository:
+
+- Go (version 1.18+)
+- make (for using Makefile configurations)
+- Docker (for using the official Docker setup files)
+
+It is recommended to work on a Unix environment, as most of the tooling is built around ready-made tools in Unix (WSL2
+for Windows / Linux / macOS).
+
+For Gno, there is no specific tooling that needs to be installed, that’s not already provided with the repo itself.
+You can utilize the `gnodev` command to facilitate Gnolang support when writing Smart Contracts in Gno, by installing it
+with `make install gnodev`.
+
+Additionally, you can also configure your editor to recognize `.gno` files as `.go` files, to get the benefit of syntax
+highlighting.
+
+Currently, we support
+an [official Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=harry-hov.gno) extension for
```

Comment:
This will come at some point, but it's too early to say it is official.
```suggestion
a [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=harry-hov.gno) extension (eventually official in the future) for
```

Reply:
Updated the wording:

[332f5bb](https://github.com/gnolang/gno/pull/469/commits/332f5bbd55581fc3f223ae67719ac62d7b3e385d)